plugin/al/usdmayautils/AL/usdmaya/utils/DiffPrimVar.cpp

```diff
@@ -24,6 +25,8 @@
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
+using namespace MayaUsdUtils;
```

Comment:
Why do you need this `using`?

Reply:
I added the using namespace instead of explicitly calling namespace with scope operator because there are so many variable and function calls to SIMD operations and I didn't want to make a bigger change than needed. 

loadu4f
f128
movemask4f
cmpne4f
loadu4d
.....

I rather get rid of the using and have the MayaUsdUtil:: for all them. 

@murphyeoin can you please comment on this?